tekton-catalog/pipeline-loops/pkg/apis/pipelineloop/v1alpha1/pipelineloop_types.go

```diff
@@ -51,6 +51,10 @@ type PipelineLoopSpec struct {
 	// IterateParam is the name of the task parameter that is iterated upon.
 	IterateParam string `json:"iterateParam"`
 
+	// The separator for IterateParam if the IterateParam is a strings with separator char, this field is optional.
+	// +optional
+	IterateParamStringSeparator string `json:"iterateParamStringSeparator,omitempty"`
```

Comment:
How about call it IterateParamSeparator?

Reply:
changed to `IterateParamSeparator `